Mathlib/Algebra/BigOperators/Ring.lean

```diff
@@ -22,6 +22,14 @@ variable {ι ι' α β γ : Type*} {κ : ι → Type*} {s s₁ s₂ : Finset ι}
 
 namespace Finset
 
+theorem sum_tsub_distrib {α β : Type*} [AddCommMonoid β] [PartialOrder β] [ExistsAddOfLE β]
```

Comment:
```suggestion
theorem sum_tsub_distrib [AddCommMonoid β] [PartialOrder β] [ExistsAddOfLE β]
```

Reply:
Actually, can you switch to using `ι` for the index type and `α` for the summand type to match the rest of the file?